OctopusDSC/Tests/cOctopusSeqLogger.Tests.ps1

```diff
@@ -97,20 +101,20 @@ try
                 }
 
                 It 'Throws an exception if Octopus is not installed and ensure is set to present' {
-                    Mock Test-Path { return $false } -ParameterFilter { $Path -eq "$($env:ProgramFiles)\Octopus Deploy\Octopus\Octopus.Server.exe" }
-                    Mock Test-Path { return $false } -ParameterFilter { $Path -eq "$($env:ProgramFiles)\Octopus Deploy\Octopus\Octopus.Server.exe.nlog" }
+                    Mock Get-OctopusServerExePath { return "" }
+                    Mock Get-TentacleExePath { return "" }
                     Mock Test-NLogDll { return $true }
                     Mock Get-NLogConfig { return  [xml] (Get-Content (Join-Path $sampleConfigPath "octopus.server.exe.nlog-with-valid-configuration-with-api-key.xml")) }
-                    { Get-TargetResourceInternal @desiredConfiguration } | Should -throw "Unable to find Octopus (checked for existence of file '$octopusServerExePath')."
+                    { Get-TargetResourceInternal @desiredConfiguration } | Should -throw "Unable to find Octopus (checked for existence of file '')."
                 }
 
                 It 'Does not throw an exception if Octopus is not installed and ensure is set to absent' {
-                    Mock Test-Path { return $false } -ParameterFilter { $Path -eq "$($env:ProgramFiles)\Octopus Deploy\Octopus\Octopus.Server.exe" }
-                    Mock Test-Path { return $false } -ParameterFilter { $Path -eq "$($env:ProgramFiles)\Octopus Deploy\Octopus\Octopus.Server.exe.nlog" }
+                    Mock Get-OctopusServerExePath { return "" }
+                    Mock Get-TentacleExePath { return "" }
                     Mock Test-NLogDll { return $true }
                     Mock Get-NLogConfig { return  [xml] (Get-Content (Join-Path $sampleConfigPath "octopus.server.exe.nlog-with-valid-configuration-with-api-key.xml")) }
                     $desiredConfiguration.Ensure = 'Absent'
-                    { Get-TargetResourceInternal @desiredConfiguration } | Should -not -throw "Unable to find Octopus (checked for existence of file '$octopusServerExePath')."
+                    { Get-TargetResourceInternal @desiredConfiguration } | Should -not -throw "Unable to find Octopus (checked for existence of file '')."
```

Comment:
Not a fan of this exception. I think if we retain our default (instead of ""), then this error will be nicer.
Either that or we change the exception to be clearer about what's going on so that it says something to the effect of "couldn't find it, looked in the registry but there was no value there" (but nicer)